src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 16px;
+  padding-top: 32px;
+  width: 200px;
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    margin: 0 0 16px;
+  }
+
+  &__label {
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    height: 40px;
+    border-radius: 5px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    color: #fff;
+    background: #00acdc;
+
+    &:hover {
+      border:  1px solid #f3f3f3;
+      color: #00acdc;
```

Comment:
1.
<img width="1012" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/cef391c0-4146-44e2-aede-7ab5f3c991d6">

2. Also the color of the border is wrong